spec/models/image_spec.rb

```diff
@@ -32,10 +32,10 @@
     expect(item).not_to be_valid
   end
 
-  xit "can be associated with an event" do
-    image1 = create(:image)
-    event = create(:event, image_id: image1.id)
-
-    expect(event.image.title).to eq(image1.title)
```

Comment:
:sparkles: Good catch! I didn't even realize I still had this skipped! Thanks! :sparkles: